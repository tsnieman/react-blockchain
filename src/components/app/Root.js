import React, { PropTypes } from 'react';

const Root = () => (
  <div style={{ color: '#8cc0b7' }}>
    <h1>Hello, World!</h1>
  </div>
);

Root.propTypes = {
  TODO: PropTypes.any,
};

export default Root;
