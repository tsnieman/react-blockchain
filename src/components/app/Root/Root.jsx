import React from 'react';

import RootRouter from 'components/app/RootRouter';

const Root = () => (
  <div>
    <RootRouter />
  </div>
);

Root.propTypes = {
};

export default Root;
