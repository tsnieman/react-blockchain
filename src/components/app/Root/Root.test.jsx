import React from 'react';
import renderer from 'react-test-renderer'; // eslint-disable-line import/no-extraneous-dependencies
import { MemoryRouter } from 'react-router';
import Root from './Root';

test('Root is tested', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Root />
    </MemoryRouter>,
  );

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
