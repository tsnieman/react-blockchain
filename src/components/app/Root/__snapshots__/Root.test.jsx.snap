exports[`test Root is tested 1`] = `
<div>
  <div>
    RootRouter
  </div>
</div>
`;
